src/features/dashboard/components/TokenRegister/index.tsx

```diff
@@ -0,0 +1,268 @@
+                label='Admin'
+                size='sm'
+                checked={isAdminChecked}
+                onChange={(e: React.ChangeEvent<HTMLInputElement>) => {
+                  setIsAdminChecked(e.target.checked);
+                  if (e.target.checked) {
+                    setIsAdminPopupVisible(true);
+                  } else {
+                    setIsAdminPopupVisible(false);
+                    setIsAdminChecked(false);
+                  }
+                }}
+              />
+              <label htmlFor='admin-scope'>
+                <Text>
+                  This scope will allow third-party apps to open accounts for you, manage your
+                  settings and token usage, and more.
+                </Text>
+              </label>
+              <SectionMessage
+                message={`Do not share tokens with the admin scope with unauthorised parties.`}
+                size='md'
+                status='warning'
+                className='mst'
+              />
+            </div>
+          </div>
+
+          {errors.admin && <span className='error-message'>{errors.admin.message}</span>}
+
+          <div className='token_register__inputfield'>
+            <div className='token_register__name'>
+              <TextField
+                {...register('token_name')}
+                label='Enter your token name'
+                placeholder='Token name'
+                inputSize='md'
+                variant='outline'
+              />
+            </div>
+            {errors?.token_name && errors?.token_name?.type === 'required' && (
+              <span className='error-message'>{errors.token_name?.message}</span>
+            )}
+            <RestrictionComponent error={errors?.token_name?.message} />
+          </div>
+          <div className='token_register__actions'>
+            <Button
+              size='lg'
+              variant='secondary'
+              color='black'
+              type='button'
+              onClick={onCancel}
+              label='Cancel'
+            />
+            <Button
+              size='lg'
+              variant='primary'
+              role='submit'
+              disabled={!isChecked || !isDirty}
+              label='Create token'
+            />
+          </div>
+        </form>
+      </FormProvider>
+
+      <Modal
+        isOpened={isAdminPopupVisible}
+        primaryButtonLabel='Enable admin access'
+        secondaryButtonLabel='Cancel'
+        primaryButtonCallback={handlePopupConfirm}
+        secondaryButtonCallback={handlePopupCancel}
+        isMobile={deviceType !== 'desktop'}
+        showSecondaryButton
+        showPrimaryButton
+        shouldCloseOnSecondaryButtonClick
+        className='admin-scope-modal'
+        showHandleBar
+        disableCloseOnOverlay={true}
+      >
+        <div className='adminScopePopup__icons'>
+          <StandaloneCircleExclamationRegularIcon fill='#C47D00' iconSize='2xl' />
```

Comment:
@agrim-deriv can we use a class here and put the color into a variable if it's not already there?

Reply:
Fixed